scripts/build-components-package.ts

```diff
@@ -0,0 +1,70 @@
+import { join } from 'path';
+import fs from 'fs-extra';
+import glob from 'glob';
+import { execa } from 'execa';
+import dirname from 'es-dirname';
+import { objFilter, deepCopyObject } from '../src/Utility/common';
+import website_pjson from '../package.json';
+import pjson_template from '../components-package/package.template.json';
+
+const website_dir = join(dirname(), '..');
+const package_dir = join(website_dir, 'components-package');
+const generated_dir = join(package_dir, 'src', 'generated');
+
+(async () => {
+    // `make clean`
+    fs.removeSync(join(package_dir, '.parcel-cache'));
+    fs.removeSync(join(package_dir, 'dist'));
+    fs.removeSync(join(package_dir, 'src', 'generated'));
+    fs.removeSync(join(package_dir, 'package.json'));
+
+    // Create `package.json` according to the template.
+    const pjson = deepCopyObject(pjson_template);
+    for (const property of [
+        'version',
+        'keywords',
+        'repository',
+        'bugs',
+        'homepage',
+        'author',
+        'contributors',
+        'license',
+    ])
+        pjson[property] = website_pjson[property];
+    const dependencies = objFilter(website_pjson.dependencies, ([pkg]) => pkg !== 'preact');
+    const peerDependencies = objFilter(website_pjson.dependencies, ([pkg]) => pkg === 'preact');
+    pjson.dependencies = dependencies;
+    pjson.peerDependencies = peerDependencies;
+    fs.writeFileSync(join(package_dir, 'package.json'), JSON.stringify(pjson, null, 4));
+
+    // Copy license and attribution files over.
+    for (const filename of ['LICENSE', 'ATTRIBUTION', 'AUTHORS'])
+        fs.copySync(join(website_dir, filename), join(package_dir, filename));
+
+    fs.ensureDirSync(generated_dir);
+
+    // Build a JSON module that exports the templates.
+    const template_dir = join(website_dir, 'static', 'templates');
```

Comment:
Maybe this should fetch the templates first? I mean, this relies on a fresh run of the deploy script…